server/service/src/sync/program_schemas/patient.json

```diff
@@ -156,30 +156,12 @@
     "Note": {
       "properties": {
         "authorId": {
-          "additionalProperties": {
-            "type": "string"
-          },
-          "properties": {
-            "length": {
-              "type": "number"
-            }
-          },
-          "required": ["length"],
-          "type": "object"
+          "type": "string"
         },
         "created": {
-          "additionalProperties": {
-            "type": "string"
-          },
           "description": "Datetime when the note has been created",
           "format": "date-time",
-          "properties": {
-            "length": {
-              "type": "number"
-            }
-          },
-          "required": ["length"],
-          "type": "object"
+          "type": "string"
```

Comment:
Unrelated, but this just corrects the "Note" type, as it was incorrect in the "programschemas" types (used "String" instead of "string). As discussed IRL, @clemens-msupply .

Reply:
interesting what it generated...